setup.py

```diff
@@ -44,7 +44,15 @@
         CUDAExtension(name='syncbn',
                       sources=['csrc/syncbn.cpp',
                                'csrc/welford.cu']))
-
+    ext_modules.append(
+        CUDAExtension(name='fused_layer_norm_cuda',
+                      sources=['apex/normalization/csrc/layer_norm_cuda.cpp',
+                               'apex/normalization/csrc/layer_norm_cuda_kernel.cu'],
+                      extra_compile_args={'cxx': ['-O3',],
+                                          'nvcc':['--gpu-architecture=sm_70', 
```

Comment:
Do you know how much perf gain hard-coding to sm_70 brings? I'd rather either not specify it, if it's not a lot, or compile for a few architectures (60,61,70) because apex is a general purpose package.

Reply:
I'll change it to compile for a few architectures.